packages/bs-flopflip/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "private": true,
+  "name": "bs-flopflip",
```

Comment:
Also, do you want to scope this package?

Reply:
Yeah. If reason-land is fine with it Id prefer to scope it. For consistency and npm tidiness. I guess ´@flopflip/bs-react-broadcast´ would be nice.